### PR TITLE
Batch for Updated by editor field

### PR DESCRIPTION
**Pull request by OlyaRybak**

### CODE REVIEW

**Comment by ffw-bot-ua** [comment]
Build comment file: 
--------------
===============
<strong>6796</strong> JSHint: profile standard file http://128.199.38.86/malmo_build76/build_reports/profilejshint.txt
Build site installed at http://128.199.38.86/malmo_build76
Behat tests: standard file http://128.199.38.86/malmo_build76/build_reports/report.html

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://128.199.38.86:8080/job/MALMO_PR_BUILDER/76/

**Comment by ffw-bot-ua** [comment]
Build comment file: 
--------------
===============
<strong>6796</strong> JSHint: profile standard file http://128.199.38.86/malmo_build77/build_reports/profilejshint.txt
Build site installed at http://128.199.38.86/malmo_build77
Behat tests: standard file http://128.199.38.86/malmo_build77/build_reports/report.html

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://128.199.38.86:8080/job/MALMO_PR_BUILDER/77/

**Comment by ffw-bot-ua** [comment]
Build comment file: 
--------------
===============
<strong>6796</strong> JSHint: profile standard file http://128.199.38.86/malmo_build78/build_reports/profilejshint.txt
Build site installed at http://128.199.38.86/malmo_build78
Behat tests: standard file http://128.199.38.86/malmo_build78/build_reports/report.html

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://128.199.38.86:8080/job/MALMO_PR_BUILDER/78/

File: drupal/profiles/mal/modules/custom/mal_content_updated_manually/mal_content_updated_manually.module [hunk 1]
```diff
@@ -23,3 +27,91 @@ function mal_content_updated_manually_views_pre_render(&$view) {
...
+  $batch = batch_content_update_manually_init();
+  batch_set($batch);
+}
+
+/**
+ * Custom batch init function.
+ */
+function batch_content_update_manually_init() {
+  $operations = array();
+  $nids = db_select('node', 'n')
```
Comment (M1r1k) [inline drupal/profiles/mal/modules/custom/mal_content_updated_manually/mal_content_updated_manually.module]: Just all nodes? Even without "type" condition?
If so use just node_load_multiple()

**Comment by ffw-bot-ua** [comment]
Build comment file: 
--------------
===============
<strong>6796</strong> JSHint: profile standard file http://128.199.38.86/malmo_build79/build_reports/profilejshint.txt
Build site installed at http://128.199.38.86/malmo_build79
Behat tests: standard file http://128.199.38.86/malmo_build79/build_reports/report.html

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://128.199.38.86:8080/job/MALMO_PR_BUILDER/79/

**Pull request merged**